src/schemes/fluid/weakly_compressible_sph/state_equations.jl

```diff
@@ -49,3 +49,65 @@ function inverse_state_equation(state_equation::StateEquationCole, pressure)
 
     return reference_density * tmp^(1 / exponent)
 end
+
+@inline sound_speed(eos) = eos.sound_speed
+
+@doc raw"""
+    StateEquationIdealGas(; gas_constant, temperature, gamma)
+Equation of state to describe the relationship between pressure and density
+of a gas using the Ideal Gas Law.
+# Keywords
+- `gas_constant`: Specific gas constant (R) for the gas, typically in J/(kg*K).
+- `temperature` : Absolute temperature of the gas in Kelvin.
+- `gamma`       : Heat-capacity ratio
+This struct calculates the pressure of a gas from its density using the formula:
```

Comment:
```suggestion

This struct calculates the pressure of a gas from its density using the formula:
```
Also, the equations below don't render correctly.